### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by HsiangNianian** [pypi/support#4813]
### PyPI Username

HsiangNianian

### Reason for request

Lost 2FA authentication app and security token, my development environment has switched from Windows to Linux, and the device verification method for 2FA is no longer applicable, and I have also forgotten my recovery code, thanks for your help :D

### Recovery Codes

- [X] I never generated or have lost access to the recovery codes for my account

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by Thespi-Brain** [comment]
Hi @HsiangNianian,

I have initiated an account recovery process and an email is being sent to the primary email address associated with your account. Please follow the instructions in that email and reply to it, and we will go from there. Thanks!

**Comment by HsiangNianian** [comment]
https://github.com/HydroRoll-Team/HydroRoll/tree/n0Mri7VhzJeHzLOH

**Comment by Thespi-Brain** [comment]
Hi @HsiangNianian,

Thanks for completing the verification process. The verification was successful! Your 2FA has been reset and a password reset has been issued. You should be able to regain access to your account after completing the password reset. Let us know if you have any more issues. Thanks.

**Comment by HsiangNianian** [comment]
Hello, @Thespi-Brain 

I'm very sorry. When I reset, I could not receive the recovery email, so I tried several more times until a 429 warning was triggered. Later I found that cloudfare blocked the no-reply@pypi.org mailbox (or outlook.com blocked it). Can I reset the number of retries? Because I currently point my current email address to the new domain name, so that I can receive emails from no-reply@pypi.org. 

Thank you very much for your understanding :D

![image](https://github.com/user-attachments/assets/b5d1cd97-e99a-4a88-add9-05e2e7165d09)
